.mockery.yaml

```diff
@@ -73,7 +73,8 @@ packages:
     config:
       dir: "{{ .InterfaceDir }}/mocks"
       all: true
-      exclude: "^UintParam$"
+      exclude:
+        - "^UintParam$"
```

Comment:
Seems like this approach doesn't work, as the mock for UintParam is generated https://github.com/agglayer/aggkit/blob/0acf52ae18b23ea073ac0fc6309c389c69432f2e/bridgeservice/mocks/mock_uint_param.go#L1

Reply:
but if we dont do this, we get an error

Reply:
```
aggkit/test|arpit/469⚡ ⇒  make generate-mocks
mockery --disable-version-string --with-expecter --exported
16 May 25 18:30 IST INF Starting mockery dry-run=false version=v2.53.3
16 May 25 18:30 IST INF Using config: /Users/arpit.temani/Workspace/Polygon/aggkit/.mockery.yaml dry-run=false version=v2.53.3
failed to discover recursive packages: failed to get package config: 1 error(s) decoding:

* 'exclude': source data must be an array or slice, got string
make: *** [generate-mocks] Error 1
```

Reply:
@Stefan-Ethernal did this work for you earlier?